README.md

```diff
@@ -94,6 +94,8 @@ For example, when you want to pass kubecolor result to grep (`kubecolor get pods
 kubectl provides [autocompletion feature](https://kubernetes.io/docs/tasks/tools/install-kubectl/#enable-kubectl-autocompletion). If you are
 already using it, you might have to configure it for kubecolor.
 
+#### Bash / Fish
```

Comment:
Did you make sure this works on fish? I don't have fish environment so I am not sure it.

Reply:
Ups, you are right, this was just an assumption, I will correct to 'Bash' only